### PR TITLE
Add customer statistic module

### PR DESCRIPTION
**Pull request by daniel-rose**

### CODE REVIEW

File: bundles/customer-statistic/src/FondOfOryx/Client/CustomerStatistic/Zed/CustomerStatisticZedStub.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    /**
+     * @param \Generated\Shared\Transfer\CustomerTransfer $customerTransfer
+     *
+     * @return \Generated\Shared\Transfer\CustomerStatisticResponseTransfer
+     */
+    public function incrementLoginCount(CustomerTransfer $customerTransfer): CustomerStatisticResponseTransfer
+    {
+        /** @var \Generated\Shared\Transfer\CustomerStatisticResponseTransfer $customerStatisticResponseTransfer */
+        $customerStatisticResponseTransfer = $this->zedRequestClient->call(
+            '/customer-statistic/gateway/increment-login-count',
```
Comment (julianzimmermann) [inline bundles/customer-statistic/src/FondOfOryx/Client/CustomerStatistic/Zed/CustomerStatisticZedStub.php:33]: could be better as const

File: bundles/customer-statistic/src/FondOfOryx/Zed/CustomerStatistic/Persistence/Propel/Mapper/CustomerStatisticMapper.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+     * @param \Generated\Shared\Transfer\CustomerStatisticTransfer $customerStatisticTransfer
+     *
+     * @return \Generated\Shared\Transfer\CustomerStatisticTransfer
+     */
+    public function mapEntityToTransfer(
+        FooCustomerStatistic $fooCustomerStatistic,
+        CustomerStatisticTransfer $customerStatisticTransfer
+    ): CustomerStatisticTransfer {
+        return $customerStatisticTransfer->fromArray(
+            $fooCustomerStatistic->toArray()
```
Comment (julianzimmermann) [inline bundles/customer-statistic/src/FondOfOryx/Zed/CustomerStatistic/Persistence/Propel/Mapper/CustomerStatisticMapper.php:41]: maybe
return $customerStatisticTransfer->fromArray(
            $fooCustomerStatistic->toArray()**, true**
        );

**Review by julianzimmermann — APPROVED** [review]
see comments

File: bundles/customer-statistic/src/FondOfOryx/Client/CustomerStatistic/Zed/CustomerStatisticZedStub.php (hunk 1 above)
Comment (daniel-rose) [reply bundles/customer-statistic/src/FondOfOryx/Client/CustomerStatistic/Zed/CustomerStatisticZedStub.php]: Good point. Thx

File: bundles/customer-statistic/src/FondOfOryx/Zed/CustomerStatistic/Persistence/Propel/Mapper/CustomerStatisticMapper.php (hunk 2 above)
Comment (daniel-rose) [reply bundles/customer-statistic/src/FondOfOryx/Zed/CustomerStatistic/Persistence/Propel/Mapper/CustomerStatisticMapper.php]: Good point. Thx.

File: bundles/customer-statistic/src/FondOfOryx/Zed/CustomerStatistic/Persistence/CustomerStatisticEntityManager.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        $this->getFactory()
+            ->createCustomerStatisticMapper()
+            ->mapTransferToEntity($customerStatisticTransfer, $fooCustomerStatistic);
+
+        $fooCustomerStatistic->save();
+
+        return $this->getFactory()
+            ->createCustomerStatisticMapper()
+            ->mapEntityToTransfer($fooCustomerStatistic, new CustomerStatisticTransfer());
+    }
```
Comment (sourcecube) [inline bundles/customer-statistic/src/FondOfOryx/Zed/CustomerStatistic/Persistence/CustomerStatisticEntityManager.php:37]: Should this stay as foo ?

**Review by sourcecube — APPROVED** [review]

File: bundles/customer-statistic/src/FondOfOryx/Zed/CustomerStatistic/Persistence/CustomerStatisticEntityManager.php (hunk 3 above)
Comment (daniel-rose) [reply bundles/customer-statistic/src/FondOfOryx/Zed/CustomerStatistic/Persistence/CustomerStatisticEntityManager.php:37]: Yes, i think it is a cool.

**Pull request merged**